EventFilter/L1TRawToDigi/plugins/implementations_stage2/GTCollections.cc

```diff
@@ -9,13 +9,15 @@ namespace l1t {
       event_.put(std::move(muonShowers_[0]), "MuonShower");
       event_.put(std::move(egammas_[0]), "EGamma");
       event_.put(std::move(etsums_[0]), "EtSum");
+      event_.put(std::move(zdcsums_[0]), "ZDCSum");
```

Comment:
Please change this string to `EtSumZDC`, here and elsewhere in this PR (see https://github.com/cms-sw/cmssw/pull/42707#discussion_r1314495801).

Reply:
This was addressed in Fixing Issues for PR 42733 (First Attempt). I did not update the zdcsums since I thought changing to etsumzdcs would be confusing but I can change that as well